charts/otel-cloud-stack/values.yaml

```diff
@@ -0,0 +1,2141 @@
+# Top level field indicating an override for fullname
+fullnameOverride: ""
+# Top level field indicating an override for the namespace
+namespaceOverride: ""
+# Top level field indicating an override for the kube cluster version
+kubeVersionOverride: ""
+
+# Top level field specifying the name of the cluster
+clusterName: ""
+
+# Extra environment variables to add to each collector, bridge and instrumentation
+extraEnvs: []
+
+# Top level field related to the OpenTelemetry Operator
+opentelemetry-operator:
+  # Field indicating whether the operator is enabled or not
+  enabled: false
+
+  # Sub-field for admission webhooks configuration
+  admissionWebhooks:
+    # Policy for handling failures
+    # Setting this allows for an installation of the otel operator at the same time as the collectors.
+    failurePolicy: "Ignore"
+
+# This is the default configuration for all collectors generated by the chart.
+# Any collectors in the `collectors` are overlayed on top of this configuration.
+defaultCollectorConfig:
```

Comment:
I think this name is misleading - it implies to me it configures the collector's configuration when it is configuring the OpenTelemetryCollector resource.  Could we name it `opentelemetry-collector`?